packages/storybook/project.json

```diff
@@ -55,6 +55,23 @@
           "command": "shx rm -rf node_modules"
         }
       }
+    },
+    "test": {
+      "executor": "@nx/playwright:playwright",
+      "outputs": ["{workspaceRoot}/dist/.playwright/packages/storybook"],
+      "options": {
+        "config": "packages/storybook/playwright.config.ts",
+        "browser": "chromium"
```

Comment:
should we add some reporters too? E.g. a HTML test reports about our component. This we could publish on GitHub for internal useage at first later on we can provide it via our landingpage too.